client/src/js/components/presentation/Navigation.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+/**
+ * @description Navigation Component gives the user access to
+ * navigate around the app
+ *
+ * @class Navigation
+ *
+ * @extends {Component}
+ */
+const Navigation = () => (
+  <div>
+    <nav className="navbar navbar-inverse fixed-top"
+      style={{ borderRadius: '0px' }}>
```

Comment:
Is there a reason for an inline styling here?

Reply:
I have removed inline styling